### ISSUE TITLE
3 Phase Breaker box

### ISSUE THREAD

**Issue by LeoLopesLage** [MarkBryanMilligan/LanternPowerMonitor#27]
Hi Mark. Excellent work! 
Would it be possible in future versions of the app to select a 3 phase breaker (3 CTs) in the space type?? 

My home has 3 phases and, because of that, I have the master breaker with 3 phases. I have an air conditioning system with 3 phases too. I know I can assign 1 CT to each phase and group them, but it would be so much cleaner to have an option with 3 phase breaker (3 CTs).
Thank you.

Leonardo

**Comment by LeoLopesLage** [comment]
Thank you!! Just opened 1.1.1 release email. Thanks for the 3 phase update.

**Comment by LeoLopesLage** [comment]
Not available in the AppStore yet.

![image](https://user-images.githubusercontent.com/91095702/186754844-44df9f11-7df1-4f84-9b4f-017a0769047d.jpeg)

**Comment by MarkBryanMilligan** [comment]
This GitHub page is for the raspberry pi service code and the web services code.  Version 1.1.1 refers to the raspberry pi service version.  The release that I just made allows the hubs to monitor the voltage of one phase and infer the voltage of phases 2 and 3.  I've added the ability to assign which phase each breaker is on to the android app, but I haven't changed the UI of the panel config to look like an EU panel (because there are quite a few styles).

I have a list of things to add to the iOS app, 3-phase config being one of them, along with billing plan config and a couple bug fixes.  I started a new day job a couple months ago, so I've been spread pretty thin lately.  It's coming though, I appreciate your patience.
